test/parallel/test-assert-objects.js

```diff
@@ -271,6 +271,16 @@ describe('Object Comparison Tests', () => {
           actual: { dataView: new Uint8Array(3) },
           expected: { dataView: new DataView(new ArrayBuffer(3)) },
         },
+        {
+          describe: 'throws when comparing Float32Array([+0.0]) with Float32Array([-0.0])',
```

Comment:
```suggestion
          description: 'throws when comparing Float32Array([+0.0]) with Float32Array([-0.0])',
```

Reply:
🤦 this must have been the autocomplete ... thanks 😄